gpflow/utilities/bijectors.py

```diff
@@ -43,8 +43,15 @@ def positive(lower: Optional[float] = None, base: Optional[str] = None) -> tfp.b
     return bijector
 
 
-def triangular() -> tfp.bijectors.Bijector:
+def triangular(positive_diag: bool = False, diag_bijector=None) -> tfp.bijectors.Bijector:
     """
     Returns instance of a triangular bijector.
+
+    :param positive_diag: if True, constrains the diagonal to be positive
+    :param diag_bijector: passed through to FillScaleTriL for positive_diag
     """
-    return tfp.bijectors.FillTriangular()
+    if positive_diag:
```

Comment:
can we not specify a default `diag_bijector` when `positive_diag` is set to True?

Reply:
Yes, `None` is using the TFP default (Softplus + Shift(1e-5)), see https://www.tensorflow.org/probability/api_docs/python/tfp/bijectors/FillScaleTriL